### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by littledan**

### CODE REVIEW

**Comment by SUPERADMIN-CREATOR** [comment]
[aumLib.log](https://github.com/tc39/ecma262/files/3585167/aumLib.log)

**Comment by ljharb** [comment]
@littledan who would be good to confirm with that this license file is accurate?

**Comment by littledan** [comment]
It would be good to confirm this with the TC39 management group, in particular I believe @bterlson is looking into IPR issues.

**Comment by ljharb** [comment]
This would replace #1088.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This repository is licensed according to Ecma International TC39's [Intellectual Property Policy](https://github.com/tc39/how-we-work/blob/master/ip.md). In particular:
+- Natural language text is licensed under [Ecma text copyright policy ](https://www.ecma-international.org/memento/Ecma%20copyright%20faq.htm).
```
Comment (mpcsh) [inline LICENSE.md]:
```suggestion
- Natural language text is licensed under [Ecma text copyright policy](https://www.ecma-international.org/memento/Ecma%20copyright%20faq.htm).
```

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+This repository is licensed according to Ecma International TC39's [Intellectual Property Policy](https://github.com/tc39/how-we-work/blob/master/ip.md). In particular:
+- Natural language text is licensed under [Ecma text copyright policy](https://www.ecma-international.org/memento/Ecma%20copyright%20faq.htm).
+- Source code is licensed under Ecma's MIT-style [Ecma International Policy on Submission, Inclusion and Licensing of Software ](https://www.ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.htm).
```
Comment (gesa) [inline LICENSE.md]:
```suggestion
- Source code is licensed under Ecma's MIT-style [Ecma International Policy on Submission, Inclusion and Licensing of Software](https://www.ecma-international.org/memento/Policies/Ecma_Policy_on_Submission_Inclusion_and_Licensing_of_Software.htm).
```

**Comment by littledan** [comment]
Given @gesa's approval, do we need anything else to land this patch?

**Pull request merged**